### ISSUE TITLE
Publish more than once

### ISSUE THREAD

**Issue by Apostolique** [nulastudio/NetCoreBeauty#4]
If I run publish once, it works:
```
dotnet publish -r win-x64 -c Release
```
But if I run the command a second time, the `.dll` files don't get moved.

It looks like as long as the `NetCoreBeauty` file that gets generated exists, the `.dll` files don't get moved.

**Comment by liesauer** [comment]
you are right, once `NetCoreBeauty` exists, it won't run any more, so the `deps.json` won't be modified twice and the `.dll` won't be moved.

**Comment by liesauer** [comment]
this is designed on purpose. the tool expects that the application should be moved or packed and when you publish the second time, the release dir should be nothing in there or be clean up, so when you cross two big different versions this can prevent some weird bugs. but how ever, this will de improved.

**Comment by Apostolique** [comment]
Good to know! I'll try to follow that move or pack workflow for my releases.

**Comment by liesauer** [comment]
fixed in [v1.1.8](https://github.com/nulastudio/NetCoreBeauty/releases/tag/v1.1.8)
two ways to force beauty:
1. `--force`
```shell
/path/to/ncbeauty --force /path/to/beauty runtimes
```
2. `ForceBeauty`
```xml
<ForceBeauty>True</ForceBeauty>
```
add into `PropertyGroup` section in your `csproj`
